src/main/java/opendcs/dao/ComputationDAO.java

```diff
@@ -875,53 +886,55 @@ public void writeComputation( DbComputation comp )
 		throws DbIoException
 	{
 		Logger.instance().debug2("writeComputation name=" + comp.getName());
```

Comment:
```suggestion
		log.trace("writeComputation name= {}", comp.getName());
```

Consider coming back with a separate PR to finish out the logging migration to slf4j .  I'm willing to help on that.

Reply:
will do, was mostly updating the sections I was using to identify issues.